Cargo.lock

```diff
@@ -1,6 +1,6 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
+version = 4
```

Comment:
I think this is due to a newer cargo version as my local one doesn't support this, we are pinned at 1.75.0 for local development